test/Test/Toml/Parsing/Unit/Common.hs

```diff
@@ -1,8 +1,42 @@
 {-# LANGUAGE FlexibleContexts #-}
 
-module Test.Toml.Parsing.Unit.Common where
+module Test.Toml.Parsing.Unit.Common
+       ( parseX
+       , failOn
+       , parseArray
+       , parseBool
+       , parseDouble
+       , parseInteger
```

Comment:
Wow, thatt's a huge list :open_mouth: Thank you for doing this careful and necessary work! :+1:

Reply:
This should wrap it up, i'll do the `neat-interpolation` package refactoring in another PR. :+1: